template/.github/workflows/test.yml

```diff
@@ -28,7 +28,16 @@ jobs:
         run: npm run lint
 
       - name: Run unit tests
-        run: npm run test
+        run: npm test -- --coverage
```

Comment:
What is the extra `--` before `--coverage`? 🤔

Reply:
Took me 1h Yesterday to figure it out, why the `test --coverage` command was working with `yarn` but was **not** working with `npm` ><

Answer: https://stackoverflow.com/questions/43046885/what-does-do-when-running-an-npm-command

Reply:
In short: when the CI config and package.json pass the `test --coverage` command from `npm` to `react-script test`:
- without `--`, `npm` will remove `--coverage` to the `react-script test` command, because it thinks that `--coverage` was an option for the `npm` command
- with `--`, `npm` will keep the `--coverage` to the `react-script test --coverage` command ;-) So it will work!

Reply:
Thanks for your explanation, I did not know that 😂

Reply:
Neither did I xD